src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,46 @@
+package racingcar;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static racingcar.Car.createDefaultCar;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import util.FakeGenerator;
+
+class CarTest {
+
+    @DisplayName("자동차 이름 유효성 검사 - 이름이 5자 초과이거나 빈 경우 에러 발생")
+    @ParameterizedTest
+    @ValueSource(strings = {"abcdef", "123456", ""})
+    void carTest1(String carName) {
+        // when&then
+        Assertions.assertThrows(IllegalArgumentException.class, () -> {
+            createDefaultCar(carName);
+        });
+    }
+
+    @DisplayName("자동차가 MINIMUM_MOVE_VALUE 이상일 때만 전진하는지 테스트")
+    @Test
+    void carTest2() {
+        //given
```

Comment:
```suggestion
        // given
```

이것도 사소하지만 한 칸 개행이 있으면 더 보기 좋을 것 같아서요 ㅎㅎ

Reply:
오 사소한지만 띄어쓰기가 있으니 더 좋은것같습니다!